doc/examples/viz_widgets.py

```diff
@@ -118,11 +118,11 @@ def text_clicked(obj, event):
 renderer.reset_clipping_range()
 
 """
-Uncomment the following lines to start the interaction.
+Uncomment the line with method start() to begin the interaction.
 """
 
-# show_manager.initialize()
-# show_manager.render()
+show_manager.initialize()
+show_manager.render()
```

Comment:
Do these still needs to be recommented, or is it just the line with `start` that will block?